sway-fmt-v2/src/items/item_use.rs

```diff
@@ -7,6 +7,7 @@ use crate::{
     },
 };
 use std::fmt::Write;
+use std::vec;
```

Comment:
Maybe we can also merge them like:

```suggestion
use std::{fmt::Write, vec};
```

Reply:
Also I think there are no problems if we do not have this `use` for `vec`, if you prefer we can also drop it.